src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.cpp

```diff
@@ -87,41 +188,133 @@ void jit_store_emitter::emit_impl(const std::vector<size_t> &in_idxs, const std:
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_store_emitter::store_dbyte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    TReg src = TReg(in_idxs[0]);
+    HReg src_h = HReg(in_idxs[0]);
+    SReg src_s = SReg(in_idxs[0]);
+    DReg src_d = DReg(in_idxs[0]);
+    XReg dst = XReg(out_idxs[0]);
+
+    switch (store_num_) {
+        case 0:
+            break;
+        case 1:
+            h->str(src_h, ptr(dst, byte_offset_));
+            break;
+        case 2:
+            h->str(src_s, ptr(dst, byte_offset_));
+            break;
+        case 3: {
+            XReg prc = XReg(aux_gpr_idxs[0]);
+            h->str(src_s, ptr(dst, byte_offset_));
+            h->add_imm(prc, dst, byte_offset_ + 2 * sizeof(uint16_t), h->X_DEFAULT_ADDR);
+            h->st1(src.h[2], ptr(prc));
+            break;
+        }
         case 4:
-            h->str(QReg(src.getIdx()), post_ptr(dst, byte_offset_));
+            h->str(src_d, ptr(dst, byte_offset_));
             break;
         default:
             OV_CPU_JIT_EMITTER_THROW("Unexpected number of elements to store.");
     }
 }
 
+template <cpu_isa_t isa>
+void jit_store_emitter::store_byte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    TReg src = TReg(in_idxs[0]);
+    BReg src_b = BReg(in_idxs[0]);
+    HReg src_h = HReg(in_idxs[0]);
+    SReg src_s = SReg(in_idxs[0]);
+    XReg dst = XReg(out_idxs[0]);
+
+    switch (store_num_) {
+        case 0:
+            break;
+        case 1:
+            h->str(src_b, ptr(dst, byte_offset_));
+            break;
+        case 2:
+            h->str(src_h, ptr(dst, byte_offset_));
+            break;
+        case 3: {
+            XReg prc = XReg(aux_gpr_idxs[0]);
+            h->str(src_h, ptr(dst, byte_offset_));
+            h->add_imm(prc, dst, byte_offset_ + 2 * sizeof(int8_t), h->X_DEFAULT_ADDR);
+            h->st1(src.b[2], ptr(prc));
+            break;
+        }
+        case 4:
+            h->str(src_s, ptr(dst, byte_offset_));
+            break;
+        default:
+            OV_CPU_JIT_EMITTER_THROW("Unexpected number of elements to store.");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_store_emitter::emit_isa(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    bool is_supported_precision = one_of(dst_prc_, ov::element::f32, ov::element::i32, ov::element::f16, ov::element::i8, ov::element::u8) &&
+                                  (src_prc_ == dst_prc_ || one_of(src_prc_, ov::element::f32, ov::element::i32));
+    OV_CPU_JIT_EMITTER_ASSERT(is_supported_precision, "Unsupported precision pair.");
+    OV_CPU_JIT_EMITTER_ASSERT(store_num_ <= static_cast<int>((get_vec_length() / dst_prc_.size())),
+                              "Unexpected number of elements to store.");
```

Comment:
Maybe I missed something.... But looks like if we have the vector with length 16 and u8 precision, I will be able to store `16 / 1 = 16` elements. But in the code I see only `0, 1, 2, 3, 4`.
Could you tell me please if you have plans to add support of loading/storing full the vector of u8 elements?

Reply:
Thanks Alexandra for the comment! For now I think 4 elements would be enough, because the intermediate precisions (the vector register precisions) we supported in load/store emitters are f32 and i32, and isa asimd only support computation for 4 elements for f32/i32. Even we load more elements for low precisions, we can not compute as much. I think extension will be needed when other isas are supported. Please feel free to have further discussions.

Reply:
Oh, absolutely! Applied. Thanks Alexandra!